test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md

```diff
@@ -83,7 +83,7 @@ cd test/scripts/backup-restore
 NS_PREFIX=redhat-rhoam ./j07-verify-rhsso-backup-and-restore.sh | tee test-output.txt
 ```
 
-4. **STS** - Reach out to SRE/QE for the `ManagedOpenShift-Support-Role` credentials in order to run the backup and restore script
+4. **STS** - Reach out to QE for the `osdCcsAdmin` credentials in order to run th
```

Comment:
Probably accidentally cut off the rest of the sentence here.

Reply:
Good catch! Will force fix

Reply:
Done